modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/recreate/consts/RecreateConstants.java

```diff
@@ -12,7 +12,8 @@
 public class RecreateConstants {
 
     public static final String PROCESS_KEY = "recreate";
-    public static final String ID = "id";
+    public static final String SERVICE_ID = "serviceId";
```

Comment:
we can add a constant named RECREATE_ERROR_MESSAGE, used to store the  error message when the order task failed

Reply:
fixed
like we discussed, use deploymentEntityHandler.getServiceDeploymentEntity(serviceId) to query resultMessage.